### ISSUE TITLE
setup saga middleware with redux-starter-kit's configureStore()

### ISSUE THREAD

**Issue by sunilb0575** [redux-saga/redux-saga#1911]
Hi team, I am working on the application which is purely redux-saga, but as the application is growing, the number of files is also growing. To solve this issue I am trying to setup redux-starter-kit to my current application.

Here is my store configuration file **index.js**

```
// import { applyMiddleware, createStore } from 'redux'
// import createSagaMiddleware from 'redux-saga'
// import { initSagas } from '../initSagas'
// import rootReducer from '../reducers'
// import { loadState, saveState } from './browserStorage'

// function configureStore () {
//   const sagaMiddleware = createSagaMiddleware()
//   const persistedState = loadState()
//   const createdStore = createStore(
//     rootReducer,
//     persistedState,
//     applyMiddleware(sagaMiddleware)
//   )
//   initSagas(sagaMiddleware)
//   return createdStore
// }

// export const store = configureStore()

// store.subscribe(() => {
//   saveState(store.getState())
// })

import { configureStore, getDefaultMiddleware } from 'redux-starter-kit'
import rootReducer from '../reducers'

export const store = configureStore({
  reducer: rootReducer,
  middleware: [...getDefaultMiddleware()]
})

```

The above commented out code is the old code. the new code which is not commented out is the new setup for redux-starter-kit. 

**initSaga.js**

```import * as sagas from './sagas'

export const initSagas = (sagaMiddleware) => (
  Object.keys(sagas).map((e) => sagas[e]).forEach(sagaMiddleware.run.bind(sagaMiddleware))
)
```

**the problem:**

when I set up the redux-starter-kit the old sagas are not working. 

**Long story short:**

How can I set up my existing redux-saga application with redux-starter-kit, without disturbing the current saga files?

Thank you in advance.

**Comment by Andarist** [comment]
You'd have to share your setup in a form of repository - that way I could quickly check what should be fixed.
